src/components/Docs/components/ForeignCoinsTable.tsx

```diff
@@ -34,13 +48,76 @@ const formatString = (str: string) => {
     .join(" ");
 };
 
+const addTokenToMetaMask = async (token: ForeignCoin & { chainName: string }) => {
+  if (!window.ethereum) {
+    alert("Please install an EVM wallet like MetaMask to add tokens to your wallet");
+    return;
+  }
+
+  try {
+    const wasAdded = await window.ethereum.request({
+      method: "wallet_watchAsset",
+      params: {
+        type: "ERC20",
+        options: {
+          address: token.zrc20_contract_address,
+          symbol: token.symbol,
+          decimals: token.decimals,
+        },
+      },
+    });
+
+    if (wasAdded) {
```

Comment:
Can this be `false` without throwing an error? I believe this method returns a boolean, so it could very well return `false`, which would return on nothing getting logged.